### ISSUE TITLE
Support for devicePubKey extension of WebAuthn

### ISSUE THREAD

**Issue by ttufts** [passwordless/docs#16]
Does passwordless.dev support the devicePubKey extension of webauthn? I don't see any information about it in the documentation, and I don't see any implementation of it in source code (after cursory search).

**Comment by abergs** [comment]
Hey! Happy to see this question surface. It's planned for a upcoming release - would you like to share more about your use case?

**Comment by abergs** [comment]
I agree that for sensitive websites, that implement a risk engine / policy, DPK will be valuable and our work to support it is already planned. But it's also worth to remember that for the majority of apps and websites, passkeys without DPK is still a huge improvement of user security.

Either way, we're in the same camp. Go DPK!

**Comment by marcinbukowczan64-prog** [comment]
[](url)https://github.com/bitwarden/passwordless-python/issues/7
